stubs/pynput/pynput/_util.pyi

```diff
@@ -67,6 +68,6 @@ class Events(Generic[_T, _AbstractListener_T]):
     def __iter__(self: Self) -> Self: ...
     def __next__(self) -> _T: ...
     def get(self, timeout: float | None = ...) -> _T | None: ...
-    def _event_mapper(self, event: Callable[_P, None]) -> Callable[_P, None]: ...
+    def _event_mapper(self, event: Callable[_P, _R]) -> Callable[_P, _R]: ...
```

Comment:
The returned callback actually returns `None`.

Reply:
I was led astray by the presence of `functools.wraps`. But that just means that the returned callback might pretend, in some situations, that it doesn't return `None`. It will always return `None` in reality -- you're right.